js/main.js

```diff
@@ -0,0 +1,113 @@
+var currentGyro = 0;
+var offsetGyro = 0;
+
+$(document).ready(function() {
+    $("#mainform").submit(function(e) {
+        e.preventDefault();
+        remove_form();
+    });
+
+    // sets a function that will be called  when the robot connects/disconnects
+    NetworkTables.addRobotConnectionListener(onRobotConnection, true);
+
+
+    // sets a function that will be called when any NetworkTables key/value changes
+    NetworkTables.addGlobalListener(onValueChanged, true);
```

Comment:
This is probably not a great idea, given that the vision system will be constantly spewing values over NetworkTables.